### PR TITLE
Fix parameter initialisation for galactic

### PR DESCRIPTION
**Pull request by Timple**
As per comment here: https://github.com/cyberbotics/webots_ros2/pull/221#issuecomment-856537375

I'm willing to expand this PR with the other controllers if these changes get accepted.

### CODE REVIEW

**Comment by bmagyar** [comment]
Happy to incorporate all of these into Foxy too!

**Comment by Timple** [comment]
Done, was actually only one other case.

Took the liberty of fixing some other compiler warnings as well.

File: joint_trajectory_controller/src/joint_trajectory_controller.cpp
Comment (bmagyar) [inline joint_trajectory_controller/src/joint_trajectory_controller.cpp:343]:
```suggestion
      RCLCPP_ERROR(logger, "Command interface type '" + interface + "' not allowed!");
```
removing change as #200 addresses this bit

File: joint_trajectory_controller/src/joint_trajectory_controller.cpp
Comment (bmagyar) [inline joint_trajectory_controller/src/joint_trajectory_controller.cpp:424]:
```suggestion
      RCLCPP_ERROR(logger, "State interface type '" + interface + "' not allowed!");
```
removing change as #200 addresses this bit

**Review by bmagyar — CHANGES_REQUESTED** [review]

**Comment by bmagyar** [comment]
@Timple if you leave editing by maintainers enabled next time I am happy to fix these up before merging. Now the ball is back in your court

**Comment by Timple** [comment]
Whups, don't remember turning that off. Might be default.

File: diff_drive_controller/src/diff_drive_controller.cpp
Comment (destogl) [inline diff_drive_controller/src/diff_drive_controller.cpp:66]: Propose to reduce the number of lines...

```suggestion
    node->declare_parameter<std::vector<std::string>>(
      "left_wheel_names", std::vector<std::string>());
```

File: diff_drive_controller/src/diff_drive_controller.cpp
Comment (destogl) [inline diff_drive_controller/src/diff_drive_controller.cpp:69]: Propose to reduce the number of lines...

```suggestion
    node->declare_parameter<std::vector<std::string>>(
      "right_wheel_names", std::vector<std::string>());
```

File: diff_drive_controller/src/diff_drive_controller.cpp
Comment (destogl) [inline diff_drive_controller/src/diff_drive_controller.cpp:89]: Propose to reduce the number of lines...

```suggestion
    node->declare_parameter<std::vector<double>>(
      "twist_covariance_diagonal", std::vector<double>());
```

**Review by destogl — APPROVED** [review]
Just a small nit-pick if you like. Otherwise good!

File: diff_drive_controller/src/diff_drive_controller.cpp
Comment (Timple) [reply diff_drive_controller/src/diff_drive_controller.cpp:66]: Ah, I simply ran `ament_uncrustify` and was under the impression there was only one valid outcome of that tool :)

**Comment by Timple** [comment]
All should be resolved now :)

**Review by bmagyar — APPROVED** [review]

**Comment by Timple** [comment]
Would you mind kicking CI again?

**Review by bmagyar — APPROVED** [review]

**Pull request merged**
